pulpcore/cli/core/context.py

```diff
@@ -38,13 +38,20 @@ class PulpArtifactContext(PulpEntityContext):
     HREF = "artifact_href"
     LIST_ID = "artifacts_list"
     READ_ID = "artifacts_read"
+    CREATE_ID = "artifacts_create"
+
+    def _create(self, file: IO[bytes]) -> Any:
+        return self.pulp_ctx.call(self.CREATE_ID, uploads={"file": file.read()})
 
     def upload(self, file: IO[bytes], chunk_size: int = 1000000, check_exists: bool = True) -> Any:
         upload_ctx = PulpUploadContext(self.pulp_ctx)
         start = 0
         size = os.path.getsize(file.name)
         sha256 = hashlib.sha256()
 
+        if chunk_size > size:
+            return self._create(file)["pulp_href"]
```

Comment:
Is it possible to pass the sha256 to this create call to add verification as with the chunked upload?